### ISSUE TITLE
CSS class intellisense hang

### ISSUE THREAD

**Issue by RaptorCZ** [madskristensen/WebEssentials2013#1760]
Hi,
I have problems with CSS class names intellisense in web forms. Many time I just want to write class name but it hangs for 1-2 minutes. Then there is combo with available classes. I have no idea where WE gets those classes, if web page (or control) doesn't have any css linked. All my css are bundled and included in other project (but it is not important). How to find out why this combo gets so long to pop up and how css class names for intellisense are gathered? Or any way how to disable this feature?

I have no example to share, because it is simply problem on whole solution of my web projects.

**Comment by madskristensen** [comment]
I don't think this is caused by Web Essentials. Can you try to disable it and see if the issue is still there?

**Comment by RaptorCZ** [comment]
I'm sure it has something to do with WE, because intellisense combo shows bootstrap icons before class names etc. And it is feature of WE. Web Forms designer itself has no info about framework specific class names.

**Comment by madskristensen** [comment]
Web Essentials doesn't add anything to WebForms (.aspx, .master, .ascx etc.). Are you sure it's not in a Razor file? If so, the issue has already been fixed.

**Comment by RaptorCZ** [comment]
No it is classic .ascx user control. I'll try to disable WE and we will see. I just read article http://madskristensen.net/post/fall-update-2014---my-year-in-review and there is "Angular and Bootstrap Intellisense icons are coming from Web Essentials", so I thought WE enhances web forms editor. I think I never had this problem before VS2013 Update 4 + WE for Update 4.

**Comment by RaptorCZ** [comment]
Ok, tried it on simple HTML page without WE and problem persists. So it seems to be a bug in WebForms editor.

**Comment by mhov** [comment]
I get very long hangs in .cshtml editor when starting a "ng-" attribute.  It hangs (unable to type futher) for about 10 seconds before showing the angular attribute intellisense.  I get the same hanging behavior waiting for css class name intellisense as well.  I have a lot of .css under several projects in this solution, so i could understand the css hanging, but the ng- attributes should be much faster.  VS2013 Update 4  + WE for Update 4.
